src/Schema/PgsqlTableSchema.php

```diff
@@ -6,7 +6,7 @@
 
 use Yiisoft\Db\Schema\TableSchema as AbstractTableSchema;
 
-class TableSchema extends AbstractTableSchema
+final class PgsqlTableSchema extends AbstractTableSchema
```

Comment:
Pgsql prefix seems to duplicate information that is already there in the namespace.

Reply:
Fixed.